index.js

```diff
@@ -301,8 +303,10 @@ module.exports = ({utPort, registerErrors, utMethod}) => class WebhookPort exten
                         }
                     });
                     const reply = (params = {}) => {
-                        const { body, code } = {...this.config.response, ...params};
-                        return h.response(body).code(code);
+                        const { body, headers, code } = {...this.config.response, ...params};
+                        const httpResponse = h.response(body).code(code);
+                        Object.entries(merge(headers, {})).forEach(([key, value]) => httpResponse.header(key, value));
```

Comment:
what is the purpose of `merge(headers, {})` ?

Reply:
In case headers is undefined, though i'm unsure now why I did it with merge.